db/migrate/20210118102533_add_cip_content.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class AddCipContent < ActiveRecord::Migration[6.1]
+  def change
+    create_table :course_years, id: :uuid do |t|
+      t.timestamps
+      t.column :is_year_one, :boolean, null: false
```

Comment:
What do we need this for?

Reply:
I think CIP content is two years in total, and there is likely going to be some logic around what lessons can be completed when. And this is the most basic thing I could think of to keep track of it.